interactions/api/gateway/client.py

```diff
@@ -149,7 +149,7 @@ def __init__(
             "timeout": 60,
             "autoclose": False,
             "compress": 0,
-            "headers": {"User-Agent": self._http._req._headers["User-Agent"]},
```

Comment:
`self._http._req._headers["User-Agent"]` is not token

![image](https://user-images.githubusercontent.com/25762802/187355229-3602439f-1eb1-4e4e-b298-ca4fbeb26fe0.png)

Reply:
Changed, thank you for pointing that out